packages/movies/src/Models/Movie.php

```diff
@@ -26,8 +29,16 @@ class Movie extends Model
         'country_id',
     ];
 
-    public function movieType()
+    public function medias()
     {
-        return $this->belongsTo(MovieType::class);
+        return $this->hasMany(Media::class, 'movie_id');
+    }
+
+    public function country(){
+        return $this->hasOne(Country::class,'id', 'country_id');
```

Comment:
Mấy cái function ni mi cài format code cho nó dễ nhìn.
Như cái ni nề

public function medias()
    {
        return $this->hasMany(Media::class, 'movie_id');
    }